virtme/commands/run.py

```diff
@@ -254,6 +254,19 @@ def quote_karg(arg: str) -> str:
     else:
         return arg
 
+# Validate name=path arguments from --disk and --blk-disk
+def sanitize_disk_args(func: str, arg: str) -> Tuple[str, str]:
+    namefile = arg.split('=', 1)
+    if len(namefile) != 2:
+        arg_fail('invalid argument to %s' % (func))
```

Comment:
Minor nit: you probably meant either `% func` or `% (func,)`, but this obviously works to.